src/github.com/getlantern/flashlight/ui/resources.go

```diff
@@ -1,7 +0,0 @@
-// This file will be overwritten by make genassets.
```

Comment:
Instead of versioning resources.go, we just generate it as part of the build.  `make clean` removes it, which allows us to treat it like a typical file-based rule.